### ISSUE TITLE
[Bug]: missing aria-describedby, a11y warning in development

### ISSUE THREAD

**Issue by brokuka** [radix-vue/shadcn-vue#695]
### Reproduction

https://stackblitz.com/edit/nuxt-starter-gvthbq?file=app.vue,pages%2Findex.vue

### Describe the bug

Using `alert-dialog` without description component and even with `aria-describedby="undefined"` triggers warnings in console

### System Info

```bash
System:
    OS: Linux 5.0 undefined
    CPU: (6) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
Binaries:
    Node: 18.20.3 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 10.2.3 - /usr/local/bin/npm
    pnpm: 8.15.6 - /usr/local/bin/pnpm
npmPackages:
    nuxt: ^3.12.4 => 3.12.4 
    radix-vue: ^1.9.2 => 1.9.2 
    shadcn-nuxt: ^0.10.4 => 0.10.4
```

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by kikky7** [comment]
Just include an empty description component below the Title and you will not get that warning.

`<AlertDialogDescription />`

I had same warnings when using Sheets (I think), so I guess it should work the same.

**Comment by brokuka** [comment]
> Just include an empty description component below the Title and you will not get that warning.
> 
> `<AlertDialogDescription />`
> 
> I had same warnings when using Sheets (I think), so I guess it should work the same.

just don't forget to hide it =)

`<AlertDialogDescription class="hidden" />`

**Comment by kikky7** [comment]
> > Just include an empty description component below the Title and you will not get that warning.
> > `<AlertDialogDescription />`
> > I had same warnings when using Sheets (I think), so I guess it should work the same.
> 
> just don't forget to hide it =)
> 
> `<AlertDialogDescription class="hidden" />`

Yes, I forgot to mention that part lol

**Comment by sadeghbarati** [comment]
This only happens in `dev` mode

**Comment by sadeghbarati** [comment]
This is the intended behavior `radix-vue` that warn will force some of us to make components with more accessibility score
